README.md

```diff
@@ -12,15 +12,6 @@ Happy birding!
 ## Script Descriptions and Usage
 In general, most of these scripts will need to be edited to add your Canary Console URL (in the form of ab1234ef.canary.tools) and your API key, which can be found in the Canary Console settings.
 
-## Ansible
```

Comment:
Why are there so many removed sections from the README.md?

Reply:
You're right, I don't know why these lines have been removed, I think it may be something inherited from the fork. I leave these lines as they were, don't worry.